Assets/UnZENity-Core/Scripts/Vob/StationaryLight.cs

```diff
@@ -243,5 +250,34 @@ private void GatherRenderers()
                 }
             }
         }
+
+        public static async Task InitializeThreadSafeLightData()
```

Comment:
This static function currently resides in a MonoBehaviour. Please consider moving it to the StationaryLightsManager as it's a static singleton function which is more aligned to our coding behaviour on a Manager, rather than a Component.

Reply:
Good shout! Requested change applied 🤠